<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FontGenWidget</class>
 <widget class="QWidget" name="FontGenWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>464</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Font Settings</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QFontComboBox" name="cboFont">
        <property name="currentFont">
         <font>
          <family>Meiryo UI</family>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Size</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spnFontSize">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>128</number>
        </property>
        <property name="value">
         <number>11</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Weight</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spnFontWeight">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Advanced</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>X Offset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spnXOffset">
            <property name="minimum">
             <number>-25</number>
            </property>
            <property name="maximum">
             <number>25</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Y Offset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spnYOffset">
            <property name="minimum">
             <number>-25</number>
            </property>
            <property name="maximum">
             <number>25</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>4</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>X Margin</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spnXMargin">
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Y Margin</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spnYMargin">
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Y Shift</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spnYShift">
        <property name="minimum">
         <number>-25</number>
        </property>
        <property name="maximum">
         <number>25</number>
        </property>
        <property name="value">
         <number>-6</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Preview</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblPreview">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>256</width>
           <height>256</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: black;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="chkBoundingBoxes">
           <property name="text">
            <string>Bounding Boxes</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkAutoRefresh">
           <property name="text">
            <string>Auto</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRefreshPreview">
           <property name="text">
            <string>Refresh</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Characters</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="txtChars">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Meiryo UI</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="locale">
          <locale language="English" country="UnitedStates"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cboFont</sender>
   <signal>currentFontChanged(QFont)</signal>
   <receiver>FontGenWidget</receiver>
   <slot>font_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnFontSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FontGenWidget</receiver>
   <slot>font_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnFontWeight</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FontGenWidget</receiver>
   <slot>font_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>495</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnXOffset</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FontGenWidget</receiver>
   <slot>adv_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnYOffset</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FontGenWidget</receiver>
   <slot>adv_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRefreshPreview</sender>
   <signal>clicked()</signal>
   <receiver>FontGenWidget</receiver>
   <slot>update_preview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>427</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtChars</sender>
   <signal>textChanged()</signal>
   <receiver>FontGenWidget</receiver>
   <slot>chars_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnXMargin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FontGenWidget</receiver>
   <slot>adv_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnYMargin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FontGenWidget</receiver>
   <slot>adv_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>537</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnYShift</sender>
   <signal>valueChanged(int)</signal>
   <receiver>FontGenWidget</receiver>
   <slot>adv_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkBoundingBoxes</sender>
   <signal>stateChanged(int)</signal>
   <receiver>FontGenWidget</receiver>
   <slot>adv_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>font_changed()</slot>
  <slot>adv_changed()</slot>
  <slot>update_preview()</slot>
  <slot>chars_changed()</slot>
 </slots>
</ui>
